packages/utils/src/account.ts

```diff
@@ -0,0 +1,63 @@
+import * as bip32 from 'bip32'
+import * as bip39 from 'bip39'
+export const CELO_DERIVATION_PATH_BASE = "m/44'/52752'/0'/0"
+
+export enum MnemonicStrength {
+  s128_12words = 128,
+  s256_24words = 256,
+}
+
+export enum MnemonicLanguages {
+  chinese_simplified,
+  chinese_traditional,
+  english,
+  french,
+  italian,
+  japanese,
+  korean,
+  spanish,
+}
+
+export function generateMnemonic(
+  strength: MnemonicStrength = MnemonicStrength.s256_24words,
+  language?: MnemonicLanguages
+): string {
+  return bip39.generateMnemonic(strength, undefined, getWordList(language))
+}
+
+export function generateKeys(
+  mnemonic: string,
+  password?: string,
+  addressIndex: number = 0
+): { privateKey: string; publicKey: string } {
+  const seed = bip39.mnemonicToSeedSync(mnemonic, password)
```

Comment:
We should test that it works with the Ledger passphrase as expected and add some words about it to the docs I think.

Reply:
I checked using the same mnemonic of a ledger that I have for testing, and it derived all the index right (at least the first 4, haha)
I would check the other way. Upload a mnemonic to the Ledger

Reply:
Ok, I verified the other way... (at least 20 minutes adding words, hahaha)
But everything works. Generates the same addresses the cli and the ledger from the mnemonic generated by the cli